client/reader/sidebar/test/promo.jsx

```diff
@@ -2,9 +2,48 @@
  * @jest-environment jsdom
  */
 
-import { shallow } from 'enzyme';
+import editorReducer from 'calypso/state/editor/reducer';
+import mediaReducer from 'calypso/state/media/reducer';
+import siteSettingsReducer from 'calypso/state/site-settings/reducer';
+import timezonesReducer from 'calypso/state/timezones/reducer';
+import uiReducer from 'calypso/state/ui/reducer';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
 import { ReaderSidebarPromo, shouldRenderAppPromo } from '../promo';
 
+const initialState = {
```

Comment:
Seems like you don't need any of these; `initialState` could just be an empty object. Any particular reason to include them here?

Reply:
You are right, I have edited it.